src/storm/solver/helper/OptimisticValueIterationHelper.cpp

```diff
@@ -132,7 +132,7 @@ template<typename ValueType>
 uint64_t IterationHelper<ValueType>::repeatedIterate(std::vector<ValueType>& x, const std::vector<ValueType>& b, ValueType precision, bool relative,
                                                      boost::optional<storage::BitVector> const& schedulerFixedForRowgroup,
                                                      boost::optional<std::vector<uint_fast64_t>> const& scheduler) {
-    return repeatedIterateInternal<false, storm::solver::OptimizationDirection::Minimize>(x, b, precision, relative);
+    return repeatedIterateInternal<false, storm::solver::OptimizationDirection::Minimize>(x, b, precision, relative, schedulerFixedForRowgroup, scheduler);
```

Comment:
These changes seem more substatial then resolving warnings. @tquatmann may also want to comment ont this?